doc/sphinx/source/projects/ad405x_iio/ad405x_iio.rst

```diff
@@ -38,48 +35,62 @@ The interface used for communicating with PC based IIO clients is either Virtual
 IIO Firmware leverages the ADI created no-os and platform driver software layers
 to communicates with IIO device.
 
-.. SDP-K1 Mbed Section
+.. note::
 
-.. include:: /source/tinyiiod/sdp_k1_mbed.rst
+   This code has been developed and tested on the SDP-K1 Controller Board with
+   Arduino headers. However, the same code can be used with minimal modifications
+   on any STM32 enabled board which has Arduino Header support on it.
 
 .. Useful links Section
 
-.. include:: /source/useful_links.rst
+.. include:: /source/useful_links_stm32.rst
 
 ====================
 Hardware Connections
 ====================
 
-Required: SDP-K1 (or alternative Mbed enabled controller board, or an STM32 board ), EVAL-AD4052 board
-and USB cable.
+Required: SDP-K1 (or an STM32 board ), EVAL-AD4052 or EVAL-AD4050 board and USB cable.
 
-Connect the EVAL-AD042 board to SDP-K1 board (or any other Mbed enabled controller 
-board or an equivalent STM32 board). Connect controller board to the PC using the USB cable. 
+Connect the EVAL-AD4052/EVAL-AD4050 board to SDP-K1 board (or an equivalent STM32 board). Connect controller board to the PC using the USB cable.
 
-   .. image:: /source/projects/ad405x_iio/ad4052_connection_diagram.png
+   .. image:: /source/projects/ad405x_iio/ad405x_connection_diagram.png
       :width: 600
 
 
 ===============
 Jumper Settings
 ===============
 
-SDP-K1 :
+**SDP-K1 :**
 
 Connect the VIO_ADJUST jumper on the SDP-K1 to 3.3V position 
 to drive SDP-K1 GPIOs at 3.3V
 
-**EVAL-AD4052:**
+**EVAL-AD4052/AD4050:**
+
+Please refer to the user guide for the jumper connections on the EVAL-AD4052/AD4050 board
+
+=======================
+Communication Interface
+=======================
+
+SDP-K1 is powered through USB connection from the computer. SDP-K1 MCU board
```

Comment:
I think we can include the previous rst for communication interface as VCOM support is present for STM32 as well now

Reply:
I think we'll have to remove mentions of mbed from the file too then?